x-pack/plugin/core/src/main/resources/ecs-core-mappings.json

```diff
@@ -0,0 +1,318 @@
+{
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "ecs_timestamp": {
+          "mapping": {
+            "ignore_malformed": false,
+            "type": "date"
+          },
+          "match": "@timestamp"
+        }
+      },
+      {
+        "message_match_only_text": {
+          "mapping": {
+            "type": "match_only_text"
+          },
+          "match": "message"
+        }
+      },
+      {
+        "ip_to_ip": {
+          "mapping": {
+            "type": "ip"
+          },
+          "match": "ip",
+          "match_mapping_type": "string"
+        }
+      },
+      {
+        "_ip_to_ip": {
+          "mapping": {
+            "type": "ip"
+          },
+          "path_match": "*_ip",
+          "match_mapping_type": "string"
+        }
+      },
+      {
+        "wildcard": {
+          "mapping": {
+            "type": "wildcard"
+          },
+          "path_match": [
+            "*.io.text",
```

Comment:
I wonder if we can simplify further by saying `*.text` is always wildcard. I'm slightly surprised it is not `match_only_text`

Reply:
>  I'm slightly surprised it is not `match_only_text`

The single ECS field that matches this pattern is [`process.io.text`](https://www.elastic.co/guide/en/ecs/8.7/ecs-process.html#field-process-io-text), which seems to be machine-generated and with potentially high cardinality, which may be the reason for this mapping 🤷‍♂️ 

>  wonder if we can simplify further by saying `*.text` is always `wildcard`

I think the above is the answer why this needs to be specific. I assume you're right and normally `text` fields would better fit full text search types